common/cuda_hip/solver/batch_bicgstab_kernels.hpp.inc

```diff
@@ -27,8 +27,9 @@ __device__ __forceinline__ void initialize(
     __syncthreads();
 
     // r = b - A*x
-    advanced_apply(static_cast<ValueType>(-1.0), mat_entry, x_shared_entry,
-                   static_cast<ValueType>(1.0), r_shared_entry);
+    gko::kernels::GKO_DEVICE_NAMESPACE::batch_single_kernels::advanced_apply(
```

Comment:
is the full qualified namespace necessary? I would have thought that only `batch_single_kernels` is necessary. Or maybe this can be simplified in the next PR.

Reply:
Yes, it should be removed in the next PR for solvers.